.travis.yml

```diff
@@ -29,7 +29,8 @@ before_install:
 
 
 install:
-  - yarn install
+  - mkdir node_modules/
+  - mkdir go-binaries/
```

Comment:
This is an attempt to work around Morula. `yarn install` is no longer needed on the root level, and is causing issues because it's running the pre-publish script `bin/cross-compile` before `./morula all bin/setup` is run.  `bin/cross-compile` uses a command `go-bindata`, which is installed in the `exo-go/bin/setup` script. We can simply run `./morula all bin/setup`, but it throws an error when ignored directories haven't been created yet (`node_modules/` and `go-binaries/` in this case).

We're working on a PR for Morula so it doesn't throw when trying to ignore non-existent directories.